hs_core/views/utils.py

```diff
@@ -59,6 +61,26 @@ def upload_from_irods(username, password, host, port, zone, irods_fnames, res_fi
         fname = os.path.basename(ifname.rstrip(os.sep))
         res_files.append(UploadedFile(file=tmpFile, name=fname, size=size))
 
+# run the update script on hyrax server via ssh session for netCDF resources on demand
+# when private netCDF resources are made public so that all links of data services
```

Comment:
@hyi I am assuming this function is executed every time the resource ```public``` attribute is reset (either True or False). Is that right?